HAB5/hab5.tex

```diff
@@ -0,0 +1,422 @@
+This in turn prepares SPEX for projects requiring this correction be done on-flight, and for other projects such as horizon detection which could ``train'' on-flight by comparing analyzed data to known IMU readings.
+To this end the synchronization of clocks may be pursued.
+Another consistent challenge has been the processing of information on a stream, without information being skipped or resampled.
+Raw and processed information should be saved for later evaluation and iteration of systems.
+
+Detailed test plans and test reports will record the method and results from rigorous testing of mechanical and electrical components, software algorithms, telemetry collection, and fault detection.
+Additional and calibrations unique to HAB5 and PLANTHAB include camera alignment; camera stability; spectral response/quantum efficiency; software processing of simulations; payload NDVI/visual functioning while on ground; an electronics functional test with specific focus on power systems and APRS reliability; electronics environmental testing; and integrated system testing.
+Time-stamping of information received by different processes may be considered.
+It is also desired that metrics be included for post-flight analysis.
+
+Among the physical requirements for the payload, it is required that the HAB vehicle and payload system be better capable of protecting from environmental factors such as humidity, winds and temperature. Specific to this payload, the alignment of cameras is a question needing further exploration. An angle and separation optimal for the combination of data acquisition and securedness is required. Vibration or displacement during flights can significantly impair or corrupt all information gathered. Fixing this hurdle is critical for future projects which may look to infer distance data with a combination of two cameras or a radar altimeter setup. While the project doesn't expect that these will all be met as results, it is firmly held that the steps taken to implement these goals as best as possible will transformatively improve SPEX's advanced payload capabilities.
+
+\subsection{Milestones}
+Development and fabrication milestones are listed in \autoref{milestones-table}.
+One key consideration in choosing a time of year to launch HAB5 with PLANTHAB is the abundance of plant life in the area where it will be launched.
+In upstate New York, plant vitality is at its highest in late spring through early fall.
+
+\begin{table}[!ht]
+  \centering
+  \caption{Milestones to design and build HAB5}
+
+\begin{minipage}{.45\textwidth}
+  \noindent
+  \begin{tabular}{@{}ll@{}}
+
+  \label{milestones-table}
+    \textbf{Task} & \textbf{Complete Date} \\
+    \midrule
+    Review and Write Up of Current Systems & September 2nd Week \\
+    Mechanical Constraints for Development & September 3rd Week \\
+    Acquistion of Data from non-Visual Sensors, Ensuring Output-Processing Compatibility & October 2nd Week \\
```

Comment:
This line gets cut off. Consider changing the tabular environment to `tabularx` and changing column definition to `@{}xl@{}`